lib/inferno/entities/test_kit.rb

```diff
@@ -0,0 +1,133 @@
+require_relative '../repositories/test_kits'
+require_relative '../repositories/test_suites'
+
+module Inferno
+  module Entities
+    # @example
+    #
+    # module USCoreTestKit
+    #   class TestKit < Inferno::Entities::TestKit
+    #     id :us_core
+    #     title 'US Core Test Kit'
+    #     description <<~DESCRIPTION
+    #       This is a big markdown description of the test kit.
+    #     DESCRIPTION
+    #     suite_ids ['us_core_v311', 'us_core_v400', 'us_core_v501', 'us_core_v610']
+    #     tags ['SMART App Launch', 'US Core']
```

Comment:
This might get a little messy; i wonder if this is something that we should categorize at the platform level.  But I'm good with this for now.

Reply:
I agree that tags might want to be defined in the platform, but I think it still makes sense to store them on the test kits. Do you want me to remove this line from the example?